admin/tests-integration/sql.service.spec.js

```diff
@@ -205,27 +205,73 @@ describe('sql.service:integration', () => {
     }
   })
 
-  xit('should store the timezone offset with the datetime value', async () => {
-    const updatedAtDate = moment('2017-12-01T15:00:00.000-08:00')
+  it('should store UTC timezone when provided with a given timezone offset as moment object', async () => {
```

Comment:
can you rename these tests to something more specific about how it normalises to UTC, such as...
`should normalise a non UTC offset to UTC when stored as moment`

Reply:
done